cmd/scale.go

```diff
@@ -314,6 +378,16 @@ func (sc *scaleCmd) run(cmd *cobra.Command, args []string) error {
 					continue
 				}
 
+				if vmss.Sku != nil {
+					currentNodeCount = int(*vmss.Sku.Capacity)
+					if int(*vmss.Sku.Capacity) == sc.newDesiredAgentCount {
+						sc.printScaleTargetEqualsExisting(currentNodeCount)
+						return nil
+					} else if int(*vmss.Sku.Capacity) > sc.newDesiredAgentCount {
+						log.Warnf("VMSS vm nodes will not be cordon/drained before scaling down!")
```

Comment:
```suggestion
						log.Warnf("VMSS VM nodes will not be cordon/drained before scaling down!")
```

Reply:
this kind of makes it sound like it's by design, even though it's a gap we're planning to fix

Reply:
for your consideration